src/library/VTKExtensions/Rendering/vtk.module

```diff
@@ -7,6 +7,7 @@ DEPENDS
   VTK::RenderingAnnotation
   VTK::RenderingOpenGL2
   VTK::RenderingVolumeOpenGL2
+  VTK::opengl
```

Comment:
why is it meeded ?

Reply:
Because `vtkF3DOpenGLGridMapper` uses an opengl function (`glDrawArrays`)